src/Command/CreateConfigCommand.php

```diff
@@ -145,23 +145,25 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     $this->contentCreator->createCsvFiles();
     $this->afterAction();
 
+    $keep_content_files = $input->getOption('keep-content-files');
```

Comment:
`$input->getOption('keep-content-files')` is already used on line 82, we can move this before that and use the new variable there as well, same is true for `$input->getOption('create-content')`, which is also used twice and can be simplified by using a variable.